catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/UdpStreamMonitor.java

```diff
@@ -363,7 +364,17 @@ public void destroy(int arg) {
 
     private void shutdown() {
         if (eventLoopGroup != null) {
-            eventLoopGroup.shutdownGracefully();
+            try {
+                eventLoopGroup.shutdownGracefully()
+                        .sync();
+                if (channelFuture != null) {
+                    channelFuture.channel()
+                            .closeFuture()
+                            .sync();
+                }
+            } catch (InterruptedException e) {
```

Comment:
That may clutter up some of this code ; both `sync()`s throw an InterruptedException (we'd have to catch it in the finally block)